app/assets/stylesheets/govuk_publishing_components/components/_devolved-nations.scss

```diff
@@ -10,3 +10,25 @@
     padding: govuk-spacing(4) govuk-spacing(6);
   }
 }
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .gem-c-devolved-nations {
+    border: 2pt solid $govuk-print-text-colour;
+    margin: 0 0 5mm;
+
+    * {
+      color: $govuk-print-text-colour !important;
+    }
+
+    .govuk-list li {
+      margin-bottom: 2mm;
+    }
+
+    .govuk-link::after {
+      display: block;
```

Comment:
It looks like the `after` text containing the link href is smaller than the actual link text (in line with your other changes, which I agree with) but it looks the same here without an explicit font size declaration. Have I remembered this wrong? Is the font size of the `after` automatically smaller anyway and we can remove those other lines, or does it need adding here as well?

Reply:
Same comment for document list, and possibly others.

Reply:
I missed setting the explicit 12pt font-size here, so it's still using the 90% font-size as determined by the design system. However both 90% and 12pt look very closely matched in size for default text. 

The reason I've set 12pt sizes for printed links is to ensure printed consistency, just in case the 90% value might calculate to something other than 12pt in unexpected scenarios.

I've set this to 12pt in this case (and will do the same for other components that have the same problem).